view.py

```diff
@@ -0,0 +1,8 @@
+from trytond.pool import PoolMeta
+
+
+class View(metaclass=PoolMeta):
+    __name__ = 'ir.ui.view'
+
+    def get_rec_name(self, name):
+        return super().get_rec_name(name) + " (%s)" % self.name
```

Comment:
Help user wich view select by internal name (in case o2m fields has view_ids defined in XML views).